### ISSUE TITLE
got nan in fine-tuning process

### ISSUE THREAD

**Issue by naotokui** [hugofloresgarcia/vampnet#8]
Hi, thanks for sharing the amazing repo! 

I tried to fine-tune the model with my dataset, but after a few epochs, I started seeing nan in the validation result. 
I tried a few different datasets but got the same results. 
Any thoughts? 

<img width="557" alt="スクリーンショット 2023-07-26 14 29 47" src="https://github.com/hugofloresgarcia/vampnet/assets/970319/06e82a64-0dcf-4bf9-b19b-4bede5af634d">

**Comment by hugofloresgarcia** [comment]
hi! 
I think the nan's in the specific accuracy bins don't mean much, it probably just means that the RNG didn't sample a batch with a noise ratio in the range 0-0.5 for that specific step.  The mean accuracy values look fine. I'd listen to the samples to check if there's an issue with training. Unconditional samples should start sounding like your target distribution upwards of 50k-80k steps. 

I think this is due to more of a display bug than a training bug, so I'll think of a better way to update the progress table than  just displaying nan! thanks for opening the issue!

Your fine-tuned models should still sound good. Let me know if there's an issue with how your samples sound through the graadio app!  Thanks for opening the issue!

**Comment by hugofloresgarcia** [comment]
closing, but feel free to reopen if this becomes an issue again!
